master/internal/config/resource_config.go

```diff
@@ -87,45 +87,51 @@ func (r *ResourceConfig) ResolveResource() error {
 		r.RootManagerInternal.setName(DefaultRMName)
 	}
 
-	// Add a default resource pool for all non resource managers.
+	// Add a default resource pool for nonslurm default resource managers.
```

Comment:
note: maybe add a TODO (multirm slurm) note here, just as a reminder to bradley/whoever works on this project next?